README.md

```diff
@@ -1,22 +1,23 @@
 This README is under construction as we work to build a new community driven high performance key-value store.
 
-This project was forked from the open source Redis project right before the transition to their new source available licenses.
+This project was forked from the open source Redis project right before the transition to their new source available licenses. For now we intend to keep the 
```

Comment:
Incomplete sentence...?

Reply:
Yeah, trying to bite off too many updates before the announcement.